src/Orchard.Cms.Web/Modules/Orchard.OpenId/JwtBearerConfigureOptions.cs

```diff
@@ -1,41 +0,0 @@
-﻿using Microsoft.AspNetCore.Builder;
```

Comment:
It's not necessary to have an `IConfigureOptions<>` implementation if you can do that from `Configure`.